halpybot/packages/utils/decorators.py

```diff
@@ -111,7 +111,7 @@ def dist_exceptions(function):
 
     @functools.wraps(function)
     async def guarded(ctx, args: List[str]):
-        if len(args) <= 1:  # Minimum Number of Args is 2.
+        if not args:
```

Comment:
sanity check: the nature of this check changed and is inconsistent with the comment. verify this is intended?

Reply:
Confirmed, comment was incorrect and nature of check has changed.